### PR TITLE
Music doesn't play in pause menu

### PR DESCRIPTION
**Pull request by AdSand**
Says in the spec that music shouldn't play on the pause menu.

- Added pause functionality in the sound manager
- Music pauses and resumes when pause is toggled

### CODE REVIEW

File: Client/SoundManager.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "SoundManager.h"
+#include "SoundManager.h"
```
Comment (DelCherub) [inline Client/SoundManager.cpp]: smh

File: Client/SoundManager.cpp [hunk 2]
```diff
@@ -28,6 +29,12 @@ void SoundManager::SM_StopSound(std::string soundName) {
 	StopSound(*sounds[soundName]);
 }
 
+void NCL::CSC8503::SoundManager::SM_PauseSound(std::string soundName)
```
Comment (DelCherub) [inline Client/SoundManager.cpp]: Can you get rid of the namespacing here please (I know its an auto generated thing but still)

**Review by DelCherub — DISMISSED** [review]
Looks fine apart from a couple cleanups.

**Review by Odycious — APPROVED** [review]

File: Client/SoundManager.cpp [hunk 3]
```diff
@@ -28,6 +28,12 @@ void SoundManager::SM_StopSound(std::string soundName) {
 	StopSound(*sounds[soundName]);
 }
 
+void SoundManager::SM_PauseSound(std::string soundName)
+{
```
Comment (St4rman) [inline Client/SoundManager.cpp]: fgilthy

**Review by St4rman — APPROVED** [review]
lgtmi

**Pull request merged**